src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1129,6 +1129,56 @@ TEST_F(HMICapabilitiesTest,
   EXPECT_FALSE(file_system::FileExists(file_cache_name_));
 }
 
+TEST_F(HMICapabilitiesTest,
+       OnCapabilityInitialized_RespondToAllPendingRAIRequestsIfTheyHold) {
+  MockApplicationManager mock_app_mngr;
+  NiceMock<event_engine_test::MockEventDispatcher> mock_dispatcher;
+  MockApplicationManagerSettings mock_application_manager_settings;
+
+  const std::string hmi_capabilities_file = "hmi_capabilities_sc2.json";
+
+  ON_CALL(mock_app_mngr, event_dispatcher())
+      .WillByDefault(ReturnRef(mock_dispatcher));
+  ON_CALL(mock_app_mngr, get_settings())
+      .WillByDefault(ReturnRef(mock_application_manager_settings));
+  EXPECT_CALL(mock_application_manager_settings, hmi_capabilities_file_name())
+      .WillRepeatedly(ReturnRef(hmi_capabilities_file));
+  EXPECT_CALL(mock_application_manager_settings,
+              hmi_capabilities_cache_file_name())
+      .WillRepeatedly(ReturnRef(file_cache_name_));
```

Comment:
@ydementieiev 
This is ON_CALL

Reply:
@SNiukalov  done [ here ](https://github.com/smartdevicelink/sdl_core/pull/3306/commits/88aa98a5d715e146c6f711fcb8219d8f6fde1873)